src/main/java/net/sf/mzmine/modules/masslistmethods/chromatogrambuilder/ChromatogramBuilderTask.java

```diff
@@ -171,6 +172,9 @@ public void run() {
 	// Add new peaklist to the project
 	project.addPeakList(newPeakList);
 
+        // Add quality parameters to peaks
+        new QualityParameters(newPeakList);

```

Comment:
I moved the calculation of the quality parameters for a peak to a separate class which I can then call instead of having to add the code to all modules generating peak lists.

The reason that I don't assign the instance to any variable is that I don't need to do anything to the instance after the first run. Should I implement it in a different way?